app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="flex min-h-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
+  <div class="w-full max-w-md space-y-8 mt-10">
+    <div>
+      <h2 class="mt-6 text-center text-3xl font-bold tracking-tight text-gray-900">Budget Buddy</h2>
+      <p class="mt-2 text-center text-sm text-gray-600">
+        <a href="#" class="font-medium text-indigo-600 hover:text-indigo-500">The key to wealth is not income, it's budgeting</a>
+      </p>
+    </div>
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+      <input type="hidden" name="remember" value="true">
+      <div class="-space-y-px rounded-md shadow-sm">
+        <div>
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "relative block w-full appearance-none rounded-none rounded-t-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm", placeholder: "Email address" %>
+        </div>
+        <div>
+          <%= f.text_field :name, autofocus: true, class: "relative block w-full appearance-none rounded-none border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm", placeholder: "Name" %>
+        </div>
+        <div class="field">
+          <%= f.password_field :password, autocomplete: "new-password", class: "relative block w-full appearance-none rounded-none border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm", placeholder: "Password (6 characters minimum)" %>
+        </div>
+        <div>
+          <%= f.password_field :password_confirmation, autocomplete: "current-password", class: "relative block w-full appearance-none rounded-none rounded-b-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm", placeholder: "Password Confirmation" %>
+        </div>
+      </div>
+      <div>
+        <%= f.submit "Sign up", class: "group relative flex w-full cursor-pointer justify-center bg-secondColor py-2 px-4 mt-6 text-sm font-medium text-white focus:outline-none focus:ring-2 focus:ring-secondColor focus:ring-offset-2" %>
+        <%= render "devise/shared/links" %>
+      </div>
+    <% end %>
+  </div>
+</div>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I know the previous reviewer didn't insist on design of different pages. But I think it would be batter if you refer the **[template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)** of all pages and then  stick yours the ones from the template such for the **Authentication page**, **Signup** && **Login** pages, **Categories Page**,...
For each page, you can find its design on the **[template](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)** with correct colors and fonts